### DISCUSSION TITLE
Adding support for eights

### DISCUSSION THREAD

**Discussion by JohannesTrussellRasch** [rxhanson/Rectangle#753] [Ideas]
Is there any possibility of adding options for eights?

This is for us ultra-widescreen users. If we got eights it would be possible to have one focused window in the middle half and then four smaller windows around it for non-important stuff.

Maybe this could be added like the ninths that can be configured through the terminal, because I realise the ultra-wide community might not be the biggest. But this would be a big benefit for us.

Thank you for a wonderful product either way!

**Comment by rxhanson** [comment]
I don't think I'll add this in myself, but see no reason to turn down a good PR for this, just like the ninths one.

**Comment by LighttheLastFlare** [comment]
As someone that just got an ultra wide screen and rectangle to manage the windows; I would love to see support for eights. It would allow for my preferred layout of quarter screen / half screen / 2x eights (stacked).

**Reply by ncharris93** [reply]
I use the same layout and came here to suggest this same thing :)

**Comment by rxhanson** [comment]
Forgot to update this discussion, but a PR by the OP got merged a while back for it. You can configure via terminal commands:
https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-extra-eighths-sizing-commands

Alternatively, there are shortcut recorders for these in [Rectangle Pro](https://rectangleapp.com/pro) (paid).

**Reply by LighttheLastFlare** [reply]
Brilliant - will have a play over the weekend. Cheers.

**Comment by Dayglowlight** [comment]
Loving the rectangle app. I too am looking for "eights" for my desktop. I am not familiar with terminal commands. So if the shortcut for the top left eighth = ctrl opt shift 1 is "defaults write com.knollsoft.Rectangle topLeftEighth -dict-add keyCode -float 18 modifierFlags -float 917504"

What is the command for subsequent eighths? Like top center left eighth = ctrl opt shift 2 - what command line should I put?

Thanks!

**Reply by JohannesTrussellRasch** [reply]
If you haven't found the link to it in the `.md` file yet here is a neat little app for getting the keycodes that you want in the commands https://apps.apple.com/us/app/key-codes/id414568915

then you just swap out what eighth you want and set the key codes you want :)
